src/plugins/d3/renderer/components/Legend.tsx

```diff
@@ -139,6 +141,13 @@ function renderLegendSymbol(args: {
                     .attr('class', className)
                     .style('stroke', color);
 
+                if (d.dashStyle) {
```

Comment:
Can `dashStyle` be undefined?

Reply:
In legend item -- yes. For example, if this is legend item for pie or for treemap, where dashStyle is not applicable